### PR TITLE
Emote audio cooldown is now shared between all audible emotes

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
all audible emotes share a cooldown now, that means, for example:
emoting `*scream` will play a sound, then emoting `*cry`, if the cooldown for `*scream` hasn't passed, will not play a sound for the `*cry`, but will still play the emote. Note - all audible emotes have a default audible cooldown of 2 seconds, unless overriden on some emotes.
## Why It's Good For The Game
- prevents people from spamming multiple audio emotes in quick succession.
- allows us to add more emote sounds without fear of players abusing them for spam.
- less audio from emotes makes the audible emotes more special and less ear-fatigue-ing
- forces players to be more considerate with what audible emote they use, increasing quality of their usage for roleplay.
## Changelog
:cl: grungussus
sound: audible emotes share the same cooldown
/:cl:

### CODE REVIEW

File: code/datums/emotes.dm [hunk 1]
```diff
@@ -100,8 +100,8 @@
 	user.log_message(msg, LOG_EMOTE)
 
 	var/tmp_sound = get_sound(user)
-	if(tmp_sound && should_play_sound(user, intentional) && TIMER_COOLDOWN_FINISHED(user, type))
-		TIMER_COOLDOWN_START(user, type, audio_cooldown)
+	if(tmp_sound && should_play_sound(user, intentional) && TIMER_COOLDOWN_FINISHED(user, "1"))
+		TIMER_COOLDOWN_START(user, "1", audio_cooldown)
```
Comment (MrMelbert) [inline code/datums/emotes.dm:104]: Why `"1"`? And not something like `"sound_emote_cooldown"`

File: code/datums/emotes.dm [hunk 2]
```diff
@@ -100,8 +100,8 @@
 	user.log_message(msg, LOG_EMOTE)
 
 	var/tmp_sound = get_sound(user)
-	if(tmp_sound && should_play_sound(user, intentional) && TIMER_COOLDOWN_FINISHED(user, type))
-		TIMER_COOLDOWN_START(user, type, audio_cooldown)
+	if(tmp_sound && should_play_sound(user, intentional) && TIMER_COOLDOWN_FINISHED(user, "1"))
```
Comment (Sadboysuss) [inline code/datums/emotes.dm:103]:
```suggestion
	if(tmp_sound && should_play_sound(user, intentional) && TIMER_COOLDOWN_FINISHED(user, "audible_emote_cooldown"))
```

File: code/datums/emotes.dm (hunk 1 above)
Comment (Sadboysuss) [inline code/datums/emotes.dm:104]:
```suggestion
		TIMER_COOLDOWN_START(user, "audible_emote_cooldown", audio_cooldown)
```

**Comment by Loafin34** [comment]
the end of LRP as we know it

**Review by MrMelbert — APPROVED** [review]
Might need to add the cooldown only for intentional emotes with this but we'll see

**Pull request merged**